console/src/components/NodeFormField.tsx

```diff
@@ -11,10 +11,12 @@ type NodeFormFieldProps = {
   formik: any;
 };
 const renderNodeFormField = ({ key, formik }: NodeFormFieldProps) => {
-  console.log({ key });
-  console.log({ formik });
+  let fieldType: string = fieldNames[key] as string;
+  if (fieldType === undefined) {
+    console.warn(`No field type found for ${key}`);
+    fieldType = 'text';
```

Comment:
fall back to the `text` field type, but also throw a warning in the console so that hopefully one of us notices and addresses it. This helped me find a couple fields I'd missed the first time.